site/docs/patterns/contact-details.mdx

```diff
@@ -18,24 +18,22 @@ Contact Details is a pattern that displays communication information—for a per
     border
   />
 
-We provide design guidance for using Salt [Avatar](/salt/components/avatar), [Button](/salt/components/button) (Icon Only Secondary), [Flex Layout](/salt/components/flex-layout) Flex layout, [Stack Layout](/salt/components/stack-layout), [Icon](/salt/components/icon), [Text](/salt/components/text), and [Tooltip](/salt/components/tooltip) components to achieve the pattern. You can also find implementation examples in the [storybook](https://storybook.saltdesignsystem.com/?path=/story/patterns-contact-details--contact-details)
+We provide design guidance for using Salt [Avatar](/salt/components/avatar), [Button](/salt/components/button), [Icon](/salt/components/icon), [Text](/salt/components/text), [Tooltip](/salt/components/tooltip) and Layout components to achieve the pattern. You can also find implementation examples in the [storybook](https://storybook.saltdesignsystem.com/?path=/story/patterns-contact-details--contact-details)
```

Comment:
We provide design guidance for using Salt [Avatar](/salt/components/avatar), [Button](/salt/components/button) (Icon Only Secondary), [Stack Layout](/salt/components/stack-layout), [Icon](/salt/components/icon), [Text](/salt/components/text), and [Tooltip](/salt/components/tooltip) components to achieve the pattern. You can also find implementation examples in the [storybook](https://storybook.saltdesignsystem.com/?path=/story/patterns-contact-details--contact-details)